packages/react/src/composite/SegmentedControl/src/components/SegmentedControlButton/SegmentedControlButton.tsx

```
@@ -0,0 +1,102 @@
+    ]
+      .concat(fullWidth && classnames.fullWidth ? classnames.fullWidth : "")
+      .filter(Boolean)
+      .join(" ");
+
+    return (
+      <As
+        {...(rest as any)}
+        className={buttonClassName}
+        onClick={handleOnClick}
+        id={`segment-${ariaID}`}
+        role="button"
```

Biome (1.9.4)

[error] The elements with the following roles can be changed to the following elements:
<button>
<input type="button">


For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)